src/test/java/teammates/test/cases/action/AdminExceptionTestActionTest.java

```diff
@@ -17,9 +22,73 @@ protected String getActionUri() {
 
     @Override
     @Test
+    @SuppressWarnings("PMD.AvoidCatchingNPE") // deliberately done for testing
     public void testExecuteAndPostProcess() {
+        final String adminUserId = "admin.user";
+        gaeSimulation.loginAsAdmin(adminUserId);
+
+        String error;
+        AdminExceptionTestAction action;
+
+        ______TS("test for AssertionError");
+        error = AssertionError.class.getSimpleName();
+        action = getAction(Const.ParamsNames.ERROR, error);
+        try {
+            action.executeAndPostProcess();
+            fail("Expected AssertionError to be thrown");
+        } catch (AssertionError ae) {
+            assertEquals(ae.getMessage(), "AssertionError Testing");
```

Comment:
I took a look at the `AdminExceptionTestAction` and realized that not all exceptions have messages. If you want to you can modify the exception messages to make them clearer, and assertEquals those exception messages here. (It would be even better if you can add an explanation for that action class to explain what it does). (and it would be even better if you can resolve the inconsistent spacing for that action class).